packages/api/src/domain/medical/document-query.ts

```diff
@@ -19,41 +17,22 @@ export type DocumentQueryProgress = {
 export const convertResult = ["success", "failed"] as const;
 export type ConvertResult = (typeof convertResult)[number];
 
-export const calculateConversionProgress = ({
```

Comment:
Moved this to another file to remove the circular dependency between Patient and this file.